### ISSUE TITLE
Keycloak seems not to be affected.. 

### ISSUE THREAD

**Issue by networksecurityvodoo** [YfryTchsGD/Log4jAttackSurface#39]
The Opensource IAM Keycloak does not seem to be affected by the log4j vulnerability.

If you check the pom.xml of the Core Build it states org.jboss.logging  is used for logging. 

Source:
 https://mvnrepository.com/artifact/org.jboss.logging/jboss-logging

**Comment by pryorda** [comment]
If you use any of these libraries there is a possibility you could be impacted :: https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core/usages

**Comment by networksecurityvodoo** [comment]
@pryorda  Thanks for the hint.

I finally found time to test my assumption. 

First I tried myself if recent Keycloak Standalone responds to the vulnerabiltity.
This wasn't the case for Registration and Login. (Screen attached)

![log4jkeycloak](https://user-images.githubusercontent.com/88483159/145729846-b140fe7b-7b76-4a28-8a07-3f02ee7154c0.png)

When i searched further I stumbled upon the official Announcement of the Keycloak Core Development Team:

https://github.com/keycloak/keycloak/discussions/9078

So it seems that your Keycloak IDP is save if you didn't use these APIs in your own SPIs. (Or somebody in an SPI provided by him.)

@YfryTchsGD  The IDM Keycloak should be added with FALSE and NOTE  "See https://github.com/keycloak/keycloak/discussions/9078"  to the List.
